src/ruby/lib/grpc/generic/active_call.rb

```diff
@@ -645,5 +666,9 @@ def initialize(wrapped)
     Operation = view_class(:cancel, :cancelled?, :deadline, :execute,
                            :metadata, :status, :start_call, :wait, :write_flag,
                            :write_flag=, :trailing_metadata)
+
+    # InterceptableView further limits access to an ActiveCall's methods
+    # for use in interceptors on the client
+    InterceptableView = view_class(:deadline, :metadata, :trailing_metadata)
```

Comment:
Whoops, sorry! Realizing that `metadata` and `trailing_metadata` are probably not useful as well, since both only show the metadata/trailers sent from the server, and won't contain anything at the time that the interceptor sees them. Can we remove these too please?